### ISSUE TITLE
Doesn't seem to work with ADetailer

### ISSUE THREAD

**Issue by Kepler-16b** [hako-mikan/sd-webui-lora-block-weight#142]
When using native img2img the block weight preset works. However using preset inside ADetailer doesn't work. The result wil be as if LoRA did not take effect.

**Comment by ema7569** [comment]
Yes same not working with adetailer in txt2img for me but work with the other face editor

**Comment by hako-mikan** [comment]
Did you add "lora_block_weight" to settings of ADetailer?

**Comment by g00dbye666** [comment]
> Did you add "lora_block_weight" to settings of ADetailer?

Hello, how do I add lora_block_weight to settings of Adetailer?

**Comment by hako-mikan** [comment]
https://github.com/Bing-su/adetailer/wiki/Settings#apply-only-selected-scripts-to-adetailer
